### DISCUSSION TITLE
Add profile image to the mention list

### DISCUSSION THREAD

**Discussion by FrazeColder** [afconsult/quill-mention#373] [Q&A] [answered]
Since the PR [#341](https://github.com/quill-mention/quill-mention/pull/341) it is not possible to add a profile image to the list like in #164. We need to create [blots](https://github.com/quill-mention/quill-mention#mention-blot-styling).

Can anyone please help me out how to implement this with Vue/Nuxt3 and give me a working example?

**Comment by csculley** [comment, marked as answer]
You should still be able to use `renderItem` in almost the same way

So instead of
```js
        renderItem(item) {
          return `<img src="${item.profileImageUrl}">${item.value}`;
        },
```
you can just do
```js
        renderItem(item) {
          const imgNode = document.createElement("img");
          imgNode.src = item.profileImageUrl;
          imgNode.innerText = item.value;
          return imgNode;
        },
```

[The index page on the docs](https://github.com/quill-mention/quill-mention/blob/6e040106a781711c6040465c26f1f22999e1187a/docs/index.html#L155) should have an example of using `renderItem` that you can reference and view on https://quill-mention.com. I'm not sure about the best way to integrate it into Vue/Nuxt3 off the top of my head, sorry

**Reply by FrazeColder** [reply]
Thanks man!
